src/collective/fingerpointing/browser/logview.pt

```diff
@@ -21,9 +21,82 @@
+      <tal:block tal:condition="python:view.available"
+                 tal:define="dummy python:view.build_audit_log()">
+        <form method="get"
+              action=""
+              tal:attributes="action python:view.base_url">
+          <div class="field">
+            <input type="hidden"
+                   name="size"
+                   value=""
+                   tal:attributes="value python:view.size" />
+            <input type="text"
+                   name="query"
+                   value=""
+                   class="searchButton"
+                   placeholder="Enter a search term."
+                   i18n:attributes="placeholder"
+                   tal:attributes="value python:view.query" />
+          </div>
+          <div class="formControls">
+            <input class="button-field context" type="submit" value="filter" i18n:attributes="value"/>
+          </div>
+        </form>
+        <p class="fingerpointing-pagination field">
+          <!-- Needs some love CSS-wise -->
+          [<a href=""
+              class="first"
+              i18n_translate=""
+              tal:omit-tag="python:not view.has_prev"
+              tal:attributes="href python:view.pagination_url('first')">first</a>]
+          [<a href="#"
+              class="previous"
+              i18n_translate=""
+              tal:omit-tag="python:not view.has_prev"
+              tal:attributes="href python:view.pagination_url('prev')">previous</a>]
+          <span class="current-page" tal:content="python:view.page">#</span>
+          [<a href="#"
+              class="last"
+              i18n_translate=""
+              tal:omit-tag="python:not view.has_next"
+              tal:attributes="href python:view.pagination_url('next')">next</a>]
+          &mdash; Size:
+          <a href="#"
+              class="size"
+              i18n_translate=""
+              tal:attributes="href python:view.pagination_url('size', 50)"
+              tal:omit-tag="python:view.size==50">50</a>
+          <a href="#"
+              class="size"
+              i18n_translate=""
+              tal:attributes="href python:view.pagination_url('size', 100)"
+              tal:omit-tag="python:view.size==100">100</a>
+          <a href="#"
+              class="size"
+              i18n_translate=""
+              tal:attributes="href python:view.pagination_url('size', 200)"
+              tal:omit-tag="python:view.size==200">200</a>
+          <a href="#"
+              class="size"
+              i18n_translate=""
+              tal:attributes="href python:view.pagination_url('size', 500)"
+              tal:omit-tag="python:view.size==500">500</a>
+          <a href="#"
+              class="size"
+              i18n_translate=""
+              tal:attributes="href python:view.pagination_url('size', 1000)"
+              tal:omit-tag="python:view.size==1000">1000</a>
+          &mdash;
+          [<a href=""
```

Comment:
this must be a real button and be placed side by side with the filter one.

Reply:
Well, a ``type="reset"`` button  needs javascript in order to delete pre-filled values. Also it needs javascript to submit it. So, we do not want it in here, right?

In Plone 5 I could make it an ``<a class="plone-btn" role="button" href="....">...</a>`` but this wont work in Plone 4. So, this then just works like the current link. Supporting Plone 4+5 is a middle-sized headache. Do you have an idea how to solve it for both?

Reply:
probably we will have to use `style` and `onclick` as theres no way to include a small bit of JS and CSS in Plonr 5... unless somebody has had the brilliant idea of somehow blocking that also ;)

Reply:
Well, in Plone 5 I can get it to look like so:
![screenshot from 2017-11-23 11 57 23](https://user-images.githubusercontent.com/157140/33169611-8e983e14-d045-11e7-959d-0abd85f78b26.png)

Same markup in Plone 4 Looks like so:

![screenshot from 2017-11-23 12 07 59](https://user-images.githubusercontent.com/157140/33170009-fc054518-d046-11e7-8c7a-3891fbaec6f8.png)